src/askui/tools/askui/askui_controller.py

```diff
@@ -1,40 +1,108 @@
 from typing import List, Literal
 import grpc
-import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2_grpc as controller_v1
-import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2 as controller_v1_pbs
+import os
 
 import time
 from PIL import Image
 
 import subprocess
 import uuid
-import os
 import time
 import sys
+import json
 import time
 import subprocess
 
 from ..utils import process_exists, wait_for_port
 from askui.reporting.report import SimpleReportGenerator
 from askui.utils import draw_point_on_image
 
+import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2_grpc as controller_v1
+import askui.tools.askui.askui_ui_controller_grpc.Controller_V1_pb2 as controller_v1_pbs
+
+def load_json_file(file_path: str) -> dict:
```

Comment:
How about moving load_json_file to a helper file? Since it's used outside the class, it makes sense to keep it separate. We could also consider exposing it for users if it's generally useful!